### PR TITLE
Contract address and sol  formatting

### PR DESCRIPTION
**Pull request by patidarmanoj10**

### CODE REVIEW

**Review by rokso — APPROVED** [review]

File: deploy/0.9.0/contracts.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "MerkleBox": "0x693c78cA4C7B061EadF65AB3e6c12448Af2b6Cfb"
```
Comment (jgarzik) [inline deploy/0.9.0/contracts.json]: Please use this format, even if only one network: https://github.com/bloq/sol-address-list/blob/master/deploy/0.9.9/contracts.json

The above ad hoc format does not give us deployment version or network.

**Review by jgarzik — CHANGES_REQUESTED** [review]

File: deploy/0.9.0/contracts.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "version": "0.9.0",
+  "networks": {
+    "mainnet": {
+      "AddressListFactory": "0x27a1E374B7176341D700A8e1AF8e1b36908E3d09"
```
Comment (jgarzik) [inline deploy/0.9.0/contracts.json]: Need to fix name of contract :)

**Review by jgarzik — CHANGES_REQUESTED** [review]

**Comment by jgarzik** [comment]
1) contract name incorrect in json
2) needs MultiTransfer contract information, which is missing?   2 contracts need to be deployed, not just 1.

File: deploy/0.9.0/contracts.json (hunk 2 above)
Comment (patidarmanoj10) [reply deploy/0.9.0/contracts.json]: Oops. Sorry about that. copy paste error.

**Review by jgarzik — APPROVED** [review]

**Pull request merged**